src/database/db.js

```diff
@@ -11,7 +11,7 @@ const connectDB = async () => {
     const db = client.db('tracking_database');
     const col = db.collection('shipments');
 
-    const query = { user_id: userID };
+    const query = { user_id: userID};
```

Comment:
I would prefer to keep the space at the end to make the style consistent. If you have installed prettier in your vscode, you could turn on 'format on save'. It will format the code automatically after you save the changes.